app/services/atmosphere/optimizer.rb

```diff
@@ -80,9 +86,13 @@ def select_tmpls_and_flavors(tmpls, options={})
     def scale(hint)
       appliance = hint[:appliance]
       quantity = hint[:quantity]
+
+      action = Action.create(appliance: appliance, action_type: :scale)
+
       optimization_strategy = appliance.optimization_strategy
       appl_manager = ApplianceVmsManager.new(appliance)
       if optimization_strategy.can_scale_manually?
+        action.log("Scaling started")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.